code/datums/mutable_appearance.dm

```diff
@@ -4,10 +4,16 @@
 
 // Mutable appearances are children of images, just so you know.
 
-/mutable_appearance/New()
-	..()
-	plane = FLOAT_PLANE // No clue why this is 0 by default yet images are on FLOAT_PLANE
-						// And yes this does have to be in the constructor, BYOND ignores it if you set it as a normal var
+#if DM_BUILD > 1642
```

Comment:
errors on the linters are bnecause of the spaces... change the space to tabs

Reply:
Hm, sometimes it gets spaces instead of tabs even when that's pressed.